lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -102,6 +102,16 @@ public function get_items( $request ) {
 		$args['post_status']          = $request['status'];
 		$args['s']                    = $request['search'];
 
+		// Set before into date query. Date query must be specified as an array of an array.
+		if ( isset( $request['before'] ) ) {
+			$args['date_query'][0]['before'] = $request['before'];
```

Comment:
To prevent a PHP notice, can we create the `date_query` array before trying to assign the value?